docs/index.rst

```diff
@@ -79,6 +78,15 @@ Seismology and geodesy
 Basemaps
 ~~~~~~~~
 
+.. cssclass:: gmtgallery
```

Comment:
Need to update the gallery after #28

Reply:
I am bit confused about how to do it. I will cancel this PR and make a new one.